themes/scss/widgets/text-elements.scss

```diff
@@ -12,16 +9,16 @@
 .CDB-Widget-textBig { // 16 size
   @include text-overflow();
   display: inline-block;
-  font-size: $sFontSize-larger;
+  font-size: $sFontSize-large;
```

Comment:
Name of variable `sFontSize-large` should be written in all lowercase letters with hyphens instead of underscores